index.js

```diff
@@ -21,6 +23,30 @@ const transformBabel = src => {
   return result;
 };
 
+
+const transformTs = (src, path) => {
```

Comment:
Of course, ts-jest might have features I'm not aware of... but not sure I saw any.